dev/run-tests.py

```diff
@@ -506,6 +506,8 @@ def main():
         test_env = "amplab_jenkins"
         # add path for Python3 in Jenkins if we're calling from a Jenkins machine
         os.environ["PATH"] = "/home/anaconda/envs/py3k/bin:" + os.environ.get("PATH")
+        # add java 7 to the environment too
+        os.environ["JAVA_7_HOME"] = "/usr/java/latest"
```

Comment:
I have a weak preference towards setting this in the Jenkins configuration scripts themselves rather than hardcoding it here. I can take care of pinning this to an explicit Java 7 JDK and can push the conf change to Jenkins.